### ISSUE TITLE
Modrinth Download - Dependency not Marked

### ISSUE THREAD

**Issue by TorchTheDragon** [copycats-plus/copycats#24]
I use Prism Launcher to download mods and as I was trying to launch the game, it crashed as I didn't have the Additional Placements mod. If you need a link to it on Modrinth to have it marked as a dependency, here it is [Additional Placements](https://modrinth.com/mod/additional-placements). Not a major issue, just inconvenient when trying to find what crashes your game lol.

**Comment by Bennyboy1695** [comment]
Do you have the crash report? As it shouldnt be crashing because of that as its not a dependency

**Comment by TorchTheDragon** [comment]
This is all I have 
![Screenshot 2024-03-13 185224](https://github.com/copycats-plus/copycats/assets/72819066/9d7148fc-e5cb-45b0-a4b5-c1ae08e43f3e)

Also, on line 26 of your gradle.properties file, you have it's CurseForge ID listed

Also, the error above disappeared after installing this with your mod and actually let me launch your game

**Comment by TorchTheDragon** [comment]
It is marked as a dependency here in the common/build.gradle https://github.com/copycats-plus/copycats/blob/multiloader/common/build.gradle#L9-L27, specifically line 24

**Comment by TorchTheDragon** [comment]
I will say though that the mod that was said to be required is actually pretty helpful lol, even if it isn't a dependency, so I am kinda glad that I had to find the mod.

**Comment by hlysine** [comment]
1. Are you sure your game crashed? The logs you showed are marked as error, not fatal. Only uncaught errors crash the game, but those are expected errors that are caught by mixins.

2. Gradle dependencies are not the same as mod dependencies. The `modCompileOnly` gradle dependency tells the Java compiler to consider source code of Additional Placements during type checking, but whether it is a true dependency depends on how Copycat+ uses the classes from Additional Placements. 

Please send the crash report (if any), or at least the entire client log. Just 2 lines of error text is not enough to identify what's happening.

**Comment by TorchTheDragon** [comment]
1. Yes I am sure the game crashed as I saw the game loading then immediately close and the Prism error log appears.
2. That is actually something I didn't know so I am sorry

I didn't have a crash report as it just crashed with no crash log, only the debug and latest logs existed, but after I opened it up with Additional Placements it had stopped the crashing.

Also, I just tried to launch without it now without me editing any other files aside from disabling Additional Placements, and suddenly it launched. Maybe it was a one time thing that just decided it wanted to crash for no reason. So I will close this as it really doesn't seem like it's your issue now, but I still wonder what was happening.
